src/python/turicreate/toolkits/drawing_classifier/_tf_drawing_classifier.py

```diff
@@ -33,6 +33,15 @@ def __init__(self, net_params, batch_size, num_classes):
         self.batch_size = batch_size
 
         self.input = _tf.placeholder(_tf.float32, [None, 28, 28, 1])
```

Comment:
If the batch size is fixed to a constant, why is there still has the shape None in the first dimension?
Like in OD the shape is fixed:
https://github.com/apple/turicreate/blob/dc07e46d4e5562f0989e8532feb735391061e170/src/python/turicreate/toolkits/object_detector/_tf_model_architecture.py#L47